ci/docker-publish.yml

```diff
@@ -6,51 +6,26 @@ name: Docker
 # documentation.
 
 on:
+  schedule:
+    - cron: '0 10 * * *' # everyday at 10am
   push:
     branches: [ $default-branch ]
-    # Publish `v1.2.3` tags as releases.
+    # Publish semver tags as releases.
```

Comment:
Not necessary `v1.2.3`. Can be also be a beta release like `v2.0.8-beta.67` which is a valid semver but will not produce a latest tag because they are updated frequently, and contain many breaking changes that are (by the author's design) not yet fit for public consumption.